src/projections/aeqd.cpp

```diff
@@ -202,25 +202,21 @@ static PJ_LP aeqd_e_inverse (PJ_XY xy, PJ *P) {          /* Ellipsoidal, inverse
     PJ_LP lp = {0.0,0.0};
     struct pj_opaque *Q = static_cast<struct pj_opaque*>(P->opaque);
     double c;
-    double azi1, azi2, s12, x2, y2, lat1, lon1, lat2, lon2;
+    double azi1, azi2, s12, lat1, lon1, lat2, lon2;
 
     if ((c = hypot(xy.x, xy.y)) < EPS10) {
         lp.phi = P->phi0;
         lp.lam = 0.;
         return (lp);
     }
     if (Q->mode == OBLIQ || Q->mode == EQUIT) {
-
-        x2 = xy.x * P->a;
-        y2 = xy.y * P->a;
         lat1 = P->phi0 / DEG_TO_RAD;
-        lon1 = P->lam0 / DEG_TO_RAD;
-        azi1 = atan2(x2, y2) / DEG_TO_RAD;
-        s12 = sqrt(x2 * x2 + y2 * y2);
+        lon1 = 0;
+        azi1 = atan2(xy.x, xy.y) / DEG_TO_RAD;
+        s12 = hypot(xy.x, xy.y);
```

Comment:
s12 quantity has been already computed at line 207 in variable c. Some cleanup is possible here

Reply:
Done, thanks for the catch.